modules/local/treesummarizedexperiment.nf

```diff
@@ -0,0 +1,76 @@
+process TREESUMMARIZEDEXPERIMENT {
+    tag "$prefix"
+    label 'process_low'
+
+    conda "bioconda::bioconductor-treesummarizedexperiment=2.10.0"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
```

Comment:
The lowecase issue is because of these lines 6 and 7.

https://depot.galaxyproject.org/singularity/bioconductor-treesummarizedexperiment%3a2.10.0--r43hdfd78af_0

Reply:
This address can also be written with lowercase to fix the issue

Reply:
Thanks, and I thought those are case sensitive (and didnt test my hypothesis) :D

Reply:
Yep, this task wasn't the most challenging one you've already tackled in this PR :D

But that is the power of team work

Reply:
Hm unfortunately that seems still broken with docker:
```
     docker: invalid reference format.
    See 'docker run --help'.
```

Reply:
I think I corrected the problem, lets see if the tests succed for 826d7b7

Reply:
Celebrated too soon, happens every time...

Ahh I might know the problem. The docker name is in format "treesummarizedexperiment:2.2.0", but in the address "%3A" represents a colon ":"

So the address should be:

https://depot.galaxyproject.org/singularity/bioconductor-treesummarizedexperiment%3a2.10.0--r43hdfd78af_0

And docker name with colon